examples/apps/cra-demo/README.md

```diff
@@ -8,7 +8,7 @@ Concepts you will learn:
 4. How to use a [SharedMap distributed data structure (DDS)](https://fluidframework.com/docs/apis/map/sharedmap/) to sync data between connected clients
 
 
-\* Just want to see the code? Jump to the [finished tutorial.](https://github.com/microsoft/FluidFramework/blob/release/0.38/examples/apps/cra-demo/src/App.tsx).
+\* Just want to see the code? Jump to the [finished tutorial.](https://github.com/microsoft/FluidFramework/blob/release/0.40/examples/apps/cra-demo/src/App.js).
```

Comment:
Yes, this isn't valid yet :D

Reply:
is there a way to link this so it automatically stays up to date with version bumps?

Reply:
```suggestion
\* Just want to see the code? Jump to the [finished tutorial.](./src/App.js).
```
I can do this....i'll need to remember to update the URL when creating the tracking issue though....better than always updating this line though in code